configs/srv.yaml

```diff
@@ -0,0 +1,2 @@
+SRV:
+  critical-blocker: "project IN ('SRVOCF', 'SRVCLI', 'SRVCOM', 'SRVKE', 'SRVKS') AND issuetype = Bug AND priority = Blocker AND Status not in (Closed,'Release Pending')"
```

Comment:
Going back to our discussions - I think that including `Status not in (Closed,'Release Pending')` for example in the query is wrong. 
What is the point of filtering these bugs out? IIRC we discussed this and agreed that the filter should de done on grafana. 
Assume a bug that was opened, found critical-blocker and resolved within the updates - this bug will not be pulled and will not be displayed. This could lead to not showing any bugs under this criteria.